package.json

```diff
@@ -16,13 +16,14 @@
     "build:mobile": "webpack --config ./config/webpack.target.mobile.js",
     "build:mobile:min": "NODE_ENV=production webpack --config ./config/webpack.target.mobile.js",
     "clean": "rm -rf ./dist",
-    "lint": "standard 'src/**/*.js'",
+    "lint": "standard 'src/**/*.js' --parser babel-eslint",
     "prebuild": "npm-run-all lint clean tx",
     "prewatch": "npm-run-all clean",
     "tx": "tx pull --all || true",
     "watch": "NODE_ENV=development webpack --config ./config/webpack.target.browser.js --watch --display-chunks",
     "watch:mobile": "NODE_ENV=development webpack --config ./config/webpack.target.mobile.js --watch --display-chunks",
-    "shasum": "shasum -a 256 dist/cozy-bar.min.js && shasum -a 256 dist/cozy-bar.min.css"
+    "shasum": "shasum -a 256 dist/cozy-bar.min.js && shasum -a 256 dist/cozy-bar.min.css",
```

Comment:
Why is this happening? What is it used for?

Reply:
And even more, how is it related to cozy-bar content insertion?

Reply:
I added this commands to be more useful when you release a new cozy-bar version in the `cozy-stack` which ask for sha256 hashes. It's not related to the cozy-bar itself but still useful.

Reply:
Here, only a comma was added.

Reply:
oooh, my bad.